site/source/docs/porting/Debugging.rst

```diff
@@ -137,6 +137,58 @@ Debug printouts can even execute arbitrary JavaScript. For example::
   }
 
 
+Printing exception messages
```

Comment:
Please write `C++ exception` here and on the next line. I think it's important to avoid confusion with JS or wasm exceptions.

Reply:
done